react/BarContextProvider/index.spec.jsx

```diff
@@ -1,21 +1,21 @@
 import React from 'react'
+import { Provider, useStore } from 'react-redux'
+import configureStore from 'redux-mock-store'
+import { render, waitFor, screen } from '@testing-library/react'
 
+import { useWebviewIntent } from 'cozy-intent'
 import CozyClient, {
   createMockClient,
   CozyProvider,
   useClient
 } from 'cozy-client'
-import configureStore from 'redux-mock-store'
-import { Provider, useStore } from 'react-redux'
-import { isFlagshipApp } from 'cozy-device-helper'
 
 import BarContextProvider from '.'
 import I18n, { useI18n, translate } from '../I18n'
 
-jest.mock('cozy-device-helper', () => ({
-  isFlagshipApp: jest.fn()
-}))
-
+window.cozy.isFlagshipApp = true
```

Comment:
it means that new we only test this component in this use case. We should test it when embedded but also when it's not.

Reply:
updated